### DISCUSSION TITLE
When I creating resourse I expect status code 201, no 200

### DISCUSSION THREAD

**Discussion by gustawdaniel** [strapi/strapi#7844] [Legacy Topics]
What do you think? Why 200 is implemented as result of POST?

Can I override it in an easy way globally?

**Comment by derrickmehaffy** [comment]
Can you open a bug report for this instead of a discussion please :smiley: as you are correct. (though largely unnecessary please do follow the bug report template.)

**Comment by lauriejim** [comment]
Hello! Strapi is using Boom to manage the response format (error and success)
The thing here is that Strapi controller functions for the `create` return an object but without setting the `status` so by default `200`.
So if you want absolutely `201`  you will have to customize the `create` functions of your controllers, to add a line of code `ctx.status = 201;`
Can you please create an issue. Thank you.
